lytics.go

```diff
@@ -54,23 +59,17 @@ type Client struct {
 // ApiResp is the core api response for all Lytics endpoints. In some instances the "Status" is returned
 // as a string rather than an int. This is a known but and will be addressed / updated.
 type ApiResp struct {
-	Status  interface{}     `json:"status"`
-	Message string          `json:"message"`
-	Meta    Meta            `json:"meta"`
-	Next    string          `json:"_next"`
-	Total   int             `json:"total"`
-	Data    json.RawMessage `json:"data"`
-}
-
-type Meta struct {
-	Format   string   `json:"name"`
-	Name     []string `json:"by_fields"`
-	ByFields []string `json:"by_fields"`
+	Status  interface{}            `json:"status"`
+	Message string                 `json:"message"`
+	Meta    map[string]interface{} `json:"meta"`
+	Next    string                 `json:"_next"`
+	Total   int                    `json:"total"`
+	Data    json.RawMessage        `json:"data"`
 }
 
 // NewLytics creates a new client instance. This contains the segment pager, segment details
 // and maintains all core data used throughout this SDK
-func NewLytics(apiKey, dataApiKey interface{}, httpclient *http.Client) *Client {
+func NewLytics(apiKey string, httpclient *http.Client) *Client {
```

Comment:
Since we're removing the `dataApiKey` we should update the readme and example docs to reflect the new number of args.

Reply:
good call, fixed.